### PR TITLE
Enable logicmonitor exporter

### PR DESCRIPTION
**Pull request by khyatigandhi6**
**Description:** <Describe what has changed.>
This is the third of 3 PRs which adds the new component, Logicmonitor exporter. This PR
enables the exporter.

**Link to tracking Issue:** #13727

**Testing:** Integration testing

**Documentation:** README.md

### CODE REVIEW

File: cmd/configschema/go.mod [hunk 1]
```diff
@@ -16,6 +16,8 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 )
 
+require github.com/logicmonitor/lm-data-sdk-go v0.7.1 // indirect
+
```
Comment (mx-psi) [inline cmd/configschema/go.mod:20]: Could you join this with the main `require` block for `// indirect` dependencies just below?

**Review by mx-psi — COMMENTED** [review]
Thanks for the PR! CI is failing with
```
Generated code is out of date, please run "make genotelcontribcol" and commit the changes in this PR.
```
Could you fix that?

File: cmd/configschema/go.mod (hunk 1 above)
Comment (khyatigandhi6) [reply cmd/configschema/go.mod:20]: sure will do that!

**Review by mx-psi — APPROVED** [review]

**Review by MovieStoreGuy — APPROVED** [review]
Looks awesome! excited to see this out there.

**Comment by khyatigandhi6** [comment]
Thank you @mx-psi @MovieStoreGuy @bogdandrutu for reviewing and getting it merged!

**Pull request merged**